services/src/apidoc.rs

```diff
@@ -63,6 +78,11 @@ use utoipa::{Modify, OpenApi};
         handlers::workflows::get_workflow_provenance_handler,
         handlers::workflows::load_workflow_handler,
         handlers::workflows::register_workflow_handler,
+        handlers::datasets::list_datasets_handler,
```

Comment:
We have a separate apidoc.rs for the pro version. Please add the handlers and types there as well.

Reply:
3979352